features/security/pom.xml

```diff
@@ -121,26 +121,6 @@
             <artifactId>security-expansion-commands</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <dependency>
```

Comment:
This pom still references the `security-sts-casclaimshandler` dependency

Reply:
Great catch.